__tests__/app.test.js

```diff
@@ -69,13 +73,18 @@ describe("/api/articles/:article_id",()=>{
             .expect(200)
                 .then(({body})=>{
                     const {article} = body
-                    expect(article.title).toBe("Living in the shadow of a great man")
-                    expect(article.topic).toBe("mitch")
-                    expect(article.author).toBe("butter_bridge")
-                    expect(article.body).toBe("I find this existence challenging")
-                    expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
-                    expect(article.votes).toBe(100)
-                    expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+
```

Comment:
any changes like this to previous tickets I would recommend doing either on their own branch or straight to main